### PR TITLE
MES-3640 - Refactor WR Page into Components 

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Removed ConductedLanguage from shared components as it's only used in one place
- 3 new Waiting Room Page components - InsuranceDeclarationComponent , ResidencyDeclarationComponent, SignatureComponent
- New candidateMock to replace the mockCandidate object we use everywhere - will create a new PR to use it everywhere
- Extract configureI18N function - will create a new PR to use it everywhere else
- Tests, Tests and More Tests

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/waiting-room/cat-b/waiting-room.cat-b.page.html [hunk 1]
```diff
@@ -26,84 +26,19 @@
...
-              <signature-area formControlName="signatureAreaCtrl" [signature]="pageState.signature$ | async"
-                retryButtonText="{{ 'waitingRoom.signatureRetry' | translate }}"
-                signHereText="{{ 'waitingRoom.signaturePrompt' | translate}}"
-                validationErrorText="{{ 'waitingRoom.signatureValidation' | translate}}"
-                [showValidText]="isCtrlDirtyAndInvalid('signatureAreaCtrl')">
-              </signature-area>
-            </ion-col>
-          </ion-row>
-        </ion-col>
-      </ion-row>
```
Comment (MarkPatt89) [inline src/pages/waiting-room/cat-b/waiting-room.cat-b.page.html:106]: Awesome that you've reduced all of this into components.

File: src/pages/waiting-room/cat-b/waiting-room.cat-b.page.ts [hunk 2]
```diff
@@ -168,12 +163,10 @@ export class WaitingRoomCatBPage extends PracticeableBasePageComponent implement
         select(getConductedLanguage),
       ),
     };
-    this.rehydrateFields();
```
Comment (MarkPatt89) [inline src/pages/waiting-room/cat-b/waiting-room.cat-b.page.ts:171]: Did you check this does not impact rehydration of the WR page in any way?

**Review by MarkPatt89 — COMMENTED** [review]
I just have one question, Tom. 
We are happy rehydration has not been impacted on the WR page? (I vaguely remember John saying more components makes rehydration easier/more manageable but just wanna check)

**Review by MarkPatt89 — APPROVED** [review]
Good work!

**Review by amobarak — APPROVED** [review]

**Pull request merged**